boards/arm/adafruit_feather_nrf52840/Kconfig

```diff
@@ -7,4 +7,5 @@ config BOARD_ENABLE_DCDC
 	bool "DCDC mode"
 	select SOC_DCDC_NRF52X
 	default y
-	depends on BOARD_ADAFRUIT_FEATHER_NRF52840
+	depends on BOARD_ADAFRUIT_FEATHER_NRF52840 \
+		|| BOARD_ADAFRUIT_FEATHER_NRF52840_SENSE
```

Comment:
Should I replace this `depends on` with an `if` block [as in the xiao_ble](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/xiao_ble/Kconfig#L6C1)? The rpi_pico doesn't seem to have a corresponding option (or a Kconfig file)